analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnchangedLocalVariablesShouldBeConst.cs

```diff
@@ -463,4 +463,13 @@ public void IndexerIndexedByLambda_CrazyCoverage()
         }
         set { }
     }
+
+    public void Message()
+    {
+        var s1 = "Test";                              // Noncompliant {{Add the 'const' modifier to 's1', and replace 'var' with 'string'.}}
+        string s2 = $"This is a {nameof(Message)}";   // Compliant - Only valid in C# 10 and above
```

Comment:
the message is confusing - what is only valid in C# 10 and above? it doesn't compile otherwise?

maybe mention that `constant string interpolation is only valid in C# 10 and above"